src/sql-parser/src/ast/defs/ddl.rs

```diff
@@ -596,7 +580,7 @@ impl AstDisplay for KafkaConnectionOptionName {
             KafkaConnectionOptionName::Broker => "BROKER",
             KafkaConnectionOptionName::Brokers => "BROKERS",
             KafkaConnectionOptionName::ProgressTopic => "PROGRESS TOPIC",
-            KafkaConnectionOptionName::SshTunnel => "SSL TUNNEL",
+            KafkaConnectionOptionName::SshTunnel => "SSH TUNNEL",
```

Comment:
Wait what. I think this means no one uses kafka with ssh tunnels because...those would fail to reparse on any restart.

Reply:
I also had the same reaction.

Reply:
@def- Great catch! Consider adding a simple c.down("materialize"), c.up("materialize") in `test/ssh-connection/mzcompose.py:workflow_basic_ssh_features`

Reply:
@philip-stoev Done. I think we should generalize that and have an automated way for each mzcompose workflow to do that, I opened an issue so we don't forget: https://github.com/MaterializeInc/materialize/issues/20503